src/sentry/api/urls.py

```diff
@@ -113,6 +112,57 @@
     TeamAlertsTriggeredIndexEndpoint,
     TeamAlertsTriggeredTotalsEndpoint,
 )
+from sentry.integrations.api.endpoints.avatar.doc_integration import DocIntegrationAvatarEndpoint
+from sentry.integrations.api.endpoints.codeowners.external_actor import (
+    ExternalTeamDetailsEndpoint,
+    ExternalTeamEndpoint,
+    ExternalUserDetailsEndpoint,
+    ExternalUserEndpoint,
+)
+from sentry.integrations.api.endpoints.integration_features import IntegrationFeaturesEndpoint
+from sentry.integrations.api.endpoints.integrations import (
+    DocIntegrationDetailsEndpoint,
+    DocIntegrationsEndpoint,
+    OrganizationConfigIntegrationsEndpoint,
+    OrganizationIntegrationDetailsEndpoint,
+    OrganizationIntegrationRequestEndpoint,
+    OrganizationIntegrationsEndpoint,
+    OrganizationPluginsConfigsEndpoint,
+    OrganizationPluginsEndpoint,
+)
+from sentry.integrations.api.endpoints.internal.integration_proxy import (
+    InternalIntegrationProxyEndpoint,
+)
+from sentry.integrations.api.endpoints.organization.organization_code_mapping_codeowners import (
```

Comment:
These endpoints could be moved up to `sentry.integrations.api.endpoints` There aren't *that* many endpoints within integrations yet.